### ISSUE TITLE
Add deep learning  

### ISSUE THREAD

**Issue by tluyben** [jezen/is-thirteen#256]
There is not enough machine learning and specifically GPU usage in this particular release which means this library, although crucial to most, if not all, of my software development adventures, prevents me from fully utilizing my computers. Please fix.

**Comment by recurrence** [comment]
**_

> `APPROVED!`

_**

**Comment by buley** [comment]
This works for me.

**Comment by hermanbergwerf** [comment]
Yes, we need something that analyzes the entire world wide web to search for new 'is thirteen' trends and hashtags. Why not consider analyzing images too? There are many ways to write thirteen. Google's recent vision API might offer an elegant solution here.

**Comment by hermanbergwerf** [comment]
Also see #224
